packages/gatsby-cli/src/reporter/index.ts

```diff
@@ -40,76 +58,103 @@ if (process.env.GATSBY_LOGGER.includes(`json`)) {
   require(`./loggers/ink`)
 }
 
-const util = require(`util`)
-const { stripIndent } = require(`common-tags`)
-const chalk = require(`chalk`)
-const { trackError } = require(`gatsby-telemetry`)
-const tracer = require(`opentracing`).globalTracer()
-
-const { getErrorFormatter } = require(`./errors`)
-const { getStore } = require(`./redux`)
-import constructError from "../structured-errors/construct-error"
-
 const errorFormatter = getErrorFormatter()
 
-import type { ActivityTracker, ActivityArgs, Reporter } from "./types"
+// eslint-disable-next-line @typescript-eslint/interface-name-prefix
+export interface ActivityTracker {
```

Comment:
you could make this an `IActivityTracker` and just do `export type ActivityTracker = IActivityTracker;`